control/trajectory_follower/include/trajectory_follower/mpc.hpp

```diff
@@ -222,21 +226,24 @@ class TRAJECTORY_FOLLOWER_PUBLIC MPC
    * @brief generate MPC matrix with trajectory and vehicle model
    * @param [in] reference_trajectory used for linearization around reference trajectory
    */
-  MPCMatrix generateMPCMatrix(const trajectory_follower::MPCTrajectory & reference_trajectory);
+  MPCMatrix generateMPCMatrix(
+    const trajectory_follower::MPCTrajectory & reference_trajectory, const float64_t max_dt);
   /**
    * @brief generate MPC matrix with trajectory and vehicle model
    * @param [in] mpc_matrix parameters matrix to use for optimization
    * @param [in] x0 initial state vector
    * @param [out] Uex optimized input vector
    */
   bool8_t executeOptimization(
-    const MPCMatrix & mpc_matrix, const Eigen::VectorXd & x0, Eigen::VectorXd * Uex);
+    const MPCMatrix & mpc_matrix, const Eigen::VectorXd & x0, Eigen::VectorXd * Uex,
+    const float64_t max_dt);
   /**
    * @brief resample trajectory with mpc resampling time
    */
   bool8_t resampleMPCTrajectoryByTime(
     float64_t start_time, const trajectory_follower::MPCTrajectory & input,
-    trajectory_follower::MPCTrajectory * output) const;
+    trajectory_follower::MPCTrajectory * output, float64_t & max_dt,
+    const geometry_msgs::msg::Pose & current_pose) const;
```

Comment:
Same here.
> For clarity, output arguments should place after the input arguments.

Reply:
fix in https://github.com/autowarefoundation/autoware.universe/pull/1171/commits/d1eea82ba0fcfac85ac9453da9b4097acf9eaffc